### DISCUSSION TITLE
[Vue warn]: (deprecation ATTR_FALSE_VALUE)

### DISCUSSION THREAD

**Discussion by liranhr** [element-plus/element-plus#11195] [Q&A]
Hi, When I use simple component (el-button for instance) the console display this message:
<img width="312" alt="image" src="https://user-images.githubusercontent.com/62378984/211257194-dc3c6d0a-3b91-45ea-9b90-4ca8d412ef31.png">
There is any solution for this issue?

**Comment by ryuhangyeong** [comment]
Providing a minimal example to confirm the problem will help us troubleshoot it.

- https://stackblitz.com/
- [Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG5jb25zdCBtc2cgPSByZWYoJ0hlbGxvIFdvcmxkIScpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8aDE+e3sgbXNnIH19PC9oMT5cbiAgPGVsLWlucHV0IHYtbW9kZWw9XCJtc2dcIiAvPlxuXG4gIDxwPlxuICAgIDxlbC1pY29uIGNvbG9yPVwidmFyKC0tZWwtY29sb3ItcHJpbWFyeSlcIj48RWxlbWVudFBsdXMgLz48L2VsLWljb24+XG4gICAgRWxlbWVudCBQbHVzIHt7IEVwVmVyc2lvbiB9fSArIFZ1ZSB7eyB2dWVWZXJzaW9uIH19XG4gIDwvcD5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

**Reply by formercoding** [reply]
I have the same problem when I using Elemet-Plus@1.0 with @vue/compat@3.0. Many ElementPlus components will lead to this problem
